mathesar_ui/src/pages/database/DatabasePage.svelte

```diff
@@ -1,18 +1,26 @@
 <script lang="ts">
```

Comment:
This file still seems a bit messy. For example, `hasOnlyPublicSchema` is unused, and there is a bunch of commented-out code. I notice that this PR is supposed to close #1648. Is there another ticket which will remain open to track the work that's still needed here?

Reply:
Done beb3d45